openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -395,40 +404,44 @@ def test_set_credit_requirement_status(self):
         self.assertEqual(len(course_requirements), 2)
 
         # Initially, the status should be None
-        req_status = api.get_credit_requirement_status(self.course_key, "staff", namespace="grade", name="grade")
-        self.assertEqual(req_status[0]["status"], None)
-        self.assertEqual(req_status[0]["order"], 0)
+        self.assert_grade_requirement_status(None, 0)
+
+        # Requirement statuses cannot be changed if a CreditRequest exists
+        credit_request, __ = CreditRequest.objects.get_or_create(
```

Comment:
If we were doing this right, I'd use a factory. In this case, I simply copied a line of code from elsewhere. `get_or_create` is overkill in this case; but, in hopes of making the release cutoff, I'm going to keep it.